requirements.txt

```diff
@@ -1,5 +1,6 @@
 browserid
```

Comment:
On second thoughts, let's do this in a separate patch as it might not just be a package name change - imports will also need updating.